common/types/chat_saved_object_attributes.ts

```diff
@@ -12,7 +12,7 @@ export interface Interaction {
   conversation_id: string;
   interaction_id: string;
   create_time: string;
-  additional_info?: Record<string, unknown>;
```

Comment:
The key-value pair does not have to be Record<string, boolean> I suppose?

Reply:
Yes, it may can also be `string`. But i assumed this could be updated when needed in future or predefined.

Reply:
![image](https://github.com/opensearch-project/dashboards-assistant/assets/13493605/1ccc7e0a-69ca-4d20-850d-31370b92bfdd)
What about `{ feedback?: SendFeedbackBody, [key: string]: unknown }`.

My intent to set value as `unknown` is letting consumers to cast to other types explicitly in their functions if they want to consume specific field and know exactly what the type those values are.

Reply:
That makes sense.